### ISSUE TITLE
Non-MMIO targets for Blackhole

### ISSUE THREAD

**Issue by pjanevskiTT** [tenstorrent/tt-umd#95]
Non-MMIO targets (multichip) are not supported for Blackhole at the moment.

Not really sure what are the limitations at the moment, if there are any. When we were bringing up new features for Blackhole in UMD, we only had single chip cards, so there was no need to enable multichip.

Not sure whether this is just an implementation thing, I think there shouldn't be any fundamental limitation - (ETH firmware possibly ?)

**Comment by abhullar-tt** [comment]
I believe that eth firmware is not ready yet.

**Comment by broskoTT** [comment]
@abhullar-tt has this changed?

**Comment by pjanevskiTT** [comment]
@broskoTT do we even have a Blackhole product which is going to require non-mmio target from the host? On p300 both chips have PCIE, galaxy as well

**Comment by abhullar-tt** [comment]
> [@broskoTT](https://github.com/broskoTT) do we even have a Blackhole product which is going to require non-mmio target from the host? On p300 both chips have PCIE, galaxy as well

No, on BH all chips will have pcie connections. I think this issue can be closed. BH would never hit code paths where host does non-MMIO device writes/reads

**Comment by pjanevskiTT** [comment]
That is what I thought, thanks Almeet
